heron/api/src/scala/SerializableBiFunction.scala

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
**General Serializable-FunctionalInterfaces Comment**
I think all Serializable Functional Interfaces(SerializableFunction, SerializableConsumer etc...) should be hidden from Scala Users. This means Scala API should just accept pure Scala functions instead of them. In the light of this, i think we do not need to have these Serializable interfaces' s Scala versions. These can be wrapped at Scala StreamletImpl level by transforming incoming `Scala Function` to related `SerializableFunctionalInterface`